environments/gcp/README.md

```diff
@@ -0,0 +1,99 @@
+# Flyte on GCP
+
+Prerequisites:
+
+- [terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli#install-terraform)
+
+Procedure:
+
+1. Create a project on GCP
+2. Run ``gcloud auth application-default login``
+3. Confirm the ID of your project:
+    ```bash
+    gcloud projects list
+    ```
+
+3 . Set the default project
+
+```bash
+gcloud config set project <YOUR_PROJECT_ID>
+```
```

Comment:
Suggestion:
My personal preference would be to not set the default project and region  via `gcloud` but instead ask the user to insert them in `locals.tf`. This way, everything is explicitly in git and there isn't suddenly a diff if another engineer runs `terraform apply` on their computer which might have a different region set.